README.md

```diff
@@ -1,10 +1,33 @@
 # Minimum GHC Installer [![Build Status](https://img.shields.io/travis/fpco/minghc.svg?style=flat)](https://travis-ci.org/fpco/minghc)
 
+## Wait, are you sure you don't want `stack`?
```

Comment:
So should we go ahead with this?

On Tue, Sep 8, 2015, 11:23 PM Elliot Cameron <notifications@github.com>
wrote:

> In README.md <https://github.com/fpco/minghc/pull/87#discussion_r38974983>
> :
>
> > @@ -1,10 +1,33 @@
> >  # Minimum GHC Installer [![Build Status](https://img.shields.io/travis/fpco/minghc.svg?style=flat)](https://travis-ci.org/fpco/minghc)
> >
> > +## Wait, are you sure you don't want `stack`?
>
> I codified stack to minimize ambiguity for someone very new to the
> Haskell ecosystem.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/fpco/minghc/pull/87/files#r38974983>.
>